autocertkit/networkconf.example

```diff
@@ -1,8 +1,8 @@
 #Interface = Network ID,VLANs
-eth0 = 0,[200,204,240]
-eth1 = 1,[200,124]
-eth2 = 0,[204,240]
-eth3 = 1,[200]
+eth0 = 0,[200,204,240],vf_driver_name,vf_driver_pkg.rpm
```

Comment:
Notes: the VF driver config is moved to here, because there may be multiple model NICs with SR-IOV feature in network config file, ACK supports to certify in one time run. So the first class option is improper to use any longer.

Reply:
This makes sense, so long as its optional - the SR-IOV test case is not a required case.

Reply:
Thanks